binaries/cuprated/src/rpc/handlers/shared.rs

```diff
@@ -128,3 +128,11 @@ pub(super) async fn get_output_distribution(
         ),
     })
 }
+
+/// Always returns an [`Error`].
+///
+/// This is a temporary function used for RPC method/endpoints
+/// that are not yet ready - it should be removed when all are ready.
+pub(super) fn not_available<T>() -> Result<T, Error> {
+    Err(anyhow!("Not available"))
+}
```

Comment:
...here.

In practice, the server returns a `404` for disabled endpoints. If we send a method that routes to `not_available` (remember all of `/json_rpc` is enabled: https://github.com/Cuprate/cuprate/pull/423#discussion_r2064976151) we get a `500`.

Any errors from `?` will return a `500` as well.